controllers/operands/networkAddons.go

```diff
@@ -113,28 +119,40 @@ func (*cnaHooks) updateSpec(req *common.HcoRequest, found *networkaddonsv1.Netwo
 	return false
 }
 
-// If deployOVS annotation doesn't exists prior the upgrade - set this annotation to true;
+// If annotation doesn't exists prior the upgrade - set this annotation to true;
 // Otherwise - remain the value as it is.
-func setDeployOvsAnnotation(req *common.HcoRequest, found *networkaddonsv1.NetworkAddonsConfig) {
+func setAnnotation(req *common.HcoRequest, found *networkaddonsv1.NetworkAddonsConfig, annotation string) {
```

Comment:
This name is too generic. Please consider renaming it.

Reply:
`setCnaoAnnotation` ?

Reply:
changed to `setCnaoAnnotation`